example/viam_robot_example_app/lib/main.dart

```diff
@@ -91,42 +54,29 @@ class _MyHomePageState extends State<MyHomePage> {
       _loading = true;
     });
 
-    Future<RobotClient> robotFut;
-
-    if (dotenv.env['ROBOT_LOCATION'] != null && dotenv.env['LOCATION_SECRET'] != null) {
-      robotFut = RobotClient.atAddress(
-        dotenv.env['ROBOT_LOCATION'] ?? '',
-        RobotClientOptions.withLocationSecret(dotenv.env['LOCATION_SECRET'] ?? ''),
-      );
-    } else if (dotenv.env['API_KEY_ID'] != null && dotenv.env['API_KEY'] != null) {
-      robotFut = RobotClient.atAddress(
-        dotenv.env['ROBOT_LOCATION'] ?? '', // or whatever default value you want
-        RobotClientOptions.withApiKey(
-          dotenv.env['API_KEY_ID'] ?? '',
-          dotenv.env['API_KEY'] ?? '',
-        ),
-      );
-    } else {
-      throw Exception('None of the required variables are defined in .env. Please see README.md for more information.');
-    }
```

Comment:
I think maybe we should keep this in, as it allows a user to interact using either the API Key or the Location Secret. Unless we want to deprecate the use of location secrets entirely?

Reply:
From my understanding all the examples of using Location Secrets have been removed from the App side, I think it makes sense to remove from SDKs too.